instructure/canvas-ios
# Create Account View layout

refs: MBL-14197
affects: Parent
release note: none

test
1. enable parentQRCodePairing on parent app
2. enable studentQRCodePairing on student app
3. open student app > settings > pair with observer > should see qr code
4. scan qr code in student app w/ device that has parent app
5. parent app should open to sign up page
6. sign up
7. try to login with new user.
